modules/main.py

```diff
@@ -192,7 +193,8 @@
             debug=keywords["d"],
             alr=keywords["alr"],
             clr=keywords["clr"],
-            gamma=keywords["g"],
+            gamma=keywords["g"]
+            # maxOnlineEpisodes=keywords["E"]
```

Comment:
We reconsidered this, as `e` and `E` have the same meaning but different significance at different tests. I was removed in my opinion.